Source/RunActivity/Viewer3D/Popups/MultiPlayerWindow.cs

```diff
@@ -0,0 +1,420 @@
+                }
+            }
+        }
+
+        /// <summary>
+        /// Retrieve a formatted list <see cref="ListLabel"/>s to be displayed as an in-browser Track Monitor.
+        /// </summary>
+        /// <param name="viewer">The Viewer to read train data from.</param>
+        /// <returns>A list of <see cref="ListLabel"/>s, one per row of the popup.</returns>
+        public IEnumerable<ListLabel> MultiPlayerWindowList(Viewer viewer)
+        {
+            //Update data
+            keyPressed = "";
+            labels = new List<ListLabel>();
+            void AddLabel(ListLabel label)
+            {
+                CheckLabel(ref label);
+            }
+            void AddSeparator() => AddLabel(new ListLabel
+            {
+                FirstCol = Viewer.Catalog.GetString("Sprtr"),
+            });
+
+            labels.Clear();
+            UpdateDataEnded = false;
+            // First Block
+            // Client and server may have a time difference.
+            var time = FormatStrings.FormatTime(viewer.Simulator.ClockTime + (MultiPlayer.MPManager.IsClient() ? MultiPlayer.MPManager.Instance().serverTimeDifference : 0));
+            AddLabel(new ListLabel
+            {
+                FirstCol = $"{Viewer.Catalog.GetString("Time")}: {time}",
+                LastCol = ""
+            });
+
+            // Separator
+            AddSeparator();
+
+            // MultiPlayer
+            if (Orts.MultiPlayer.MPManager.IsMultiPlayer())
+            {
+                var text = Orts.MultiPlayer.MPManager.Instance().GetOnlineUsersInfo();
+                var multiPlayerStatus = Orts.MultiPlayer.MPManager.IsServer()
+                    ? $"{Viewer.Catalog.GetString("Dispatcher")} ({Orts.MultiPlayer.MPManager.Server.UserName})" : Orts.MultiPlayer.MPManager.Instance().AmAider
+                    ? Viewer.Catalog.GetString("Helper") : Orts.MultiPlayer.MPManager.IsClient()
+                    ? $"{Viewer.Catalog.GetString("Client")} ({Orts.MultiPlayer.MPManager.Client.UserName})" : "";
+
+                var status = $"{Viewer.Catalog.GetString("Status")}: {multiPlayerStatus}";
+
+                AddLabel(new ListLabel
+                {
+                    FirstCol = status,
+                    LastCol = ""
+                });
+
+                AddLabel(new ListLabel());
+
+                // Number of player and trains
+                foreach (var t in text.Split('\t'))
+                {
+                    AddLabel(new ListLabel
+                    {
+                        FirstCol = $"{t}",
+                        LastCol = ""
+                    });
+                }
+                AddLabel(new ListLabel());
+            }
+            else if(Orts.MultiPlayer.MPManager.Simulator.Confirmer != null)
+            {
+                var status = $"{Viewer.Catalog.GetString("Status")}: {MPManager.Catalog.GetString("Connection to the server is lost, will play as single mode")}";
```

Comment:
During initialization in timetable mode, this line is executed and triggers an exception because MPManager.Catalog is null.

Reply:
Thank you very much for detecting it so quickly.
A possible solution would be the following:

protected internal override void Initialize()
{
base.Initialize();
// Reset window size
**if( !Owner.Viewer.Simulator.TimetableMode )**
UpdateWindowSize();
}

I will generate a bug report and a PR.